.github/codeql/codeql-config.yml

```diff
@@ -0,0 +1,26 @@
+name: "ospac-simcore CodeQL config"
+
+disable-default-queries: false
+#queries:
+#  - uses: security-and-quality
+
+paths:
+  - packages/dask-task-models-library/src
+  - packages/models-library/src/models_library
+  - packages/postgres-database/src/simcore_postgres_database
+  - packages/service-integration/src/service_integration
+  - packages/service-library/src/servicelib
+  - packages/settings-library/src/settings_library
+  - packages/simcore-sdk/src/simcore_sdk
+  - services/api-server/src/simcore_service_api_server
+  - services/catalog/src/simcore_service_catalog
+  - services/dask-sidecar/src/simcore_service_dask_sidecar
+  - services/datcore-adapter/src/simcore_service_datcore_adapter
+  - services/director-v2/src/simcore_service_director_v2
+  - services/dynamic-sidecar/src/simcore_service_dynamic_sidecar
+  - services/storage/src/simcore_service_storage
+  - services/web/server/src/simcore_service_webserver
+  - "**/src"
+  - "**/source"
+paths-ignore:
```

Comment:
so I guess we never care about tests

Reply:
we do care ( al ot) about tests :-). ... but yes, I do not see how security rules would be useful in those